service/src/main/java/edu/tamu/catalog/service/FolioCatalogService.java

```diff
@@ -276,4 +285,63 @@ private void buildCoreItem(String instanceId, String barcode, NodeList nodes, Ma
         catalogItems.put(barcode, itemData);
     }
 
+    @Override
+    public FeesFines getFeesFines(String uin) throws ParseException {
+        String path = "patron/account/";
```

Comment:
I think this should be just `/patron` or `patron`. And be sure to add `/oai` or `oai` to the path for the holdings requests.

Reply:
I believe this (`patron/account`) is the FOLIO endpoint: https://s3.amazonaws.com/foliodocs/api/edge-patron/edge-patron.html#